code/modules/research/techweb/nodes/security_nodes.dm

```diff
@@ -8,6 +8,7 @@
 		"toygun",
 		"c38_rubber",
 		"c38_rubber_mag",
+		"c38",
```

Comment:
I think this means c38 appears twice?

Reply:
Actually not sure why this is here, its autolathe-exclusive and roundstart availible, this does nothing